utils/find-wasm.js

```diff
@@ -0,0 +1,35 @@
+const util = require('util');
+const glob = util.promisify(require('glob'));
+const fs = require('fs');
+
+
+
+async function findWasm(target) {
+    let res = await glob(`**/${target}/*.wasm`, { 
```

Comment:
Does this pattern match `contract/rust/target/wasm32-unknown-uknown/debug/whatever.wasm`? If it matches the behavior I get in bash on a Mac, it does not.

Reply:
Hmm I thought ** was recursive.  If you do `**/*/${target}/*.wasm` does it work?

Reply:
I just added tests that show this does work.